testutils/clustermanager/gke_test.go

```diff
@@ -273,15 +297,15 @@ func TestSetup(t *testing.T) {
 			return oldEnvFunc(s)
 		}
 		c := GKEClient{}
-		co := c.Setup(data.numNodes, data.nodeType, data.region, data.zone, data.project)
-		errPrefix := fmt.Sprintf("testing setup with:\n\tnumNodes: %v\n\tnodeType: %v\n\tregion: %v\n\tone: %v\n\tproject: %v\n\tregionEnv: %v\n\tbackupRegionEnv: %v",
+		co := c.Setup(data.numNodes, data.nodeType, data.region, data.zone, data.project, data.addons)
+		errMsg := fmt.Sprintf("testing setup with:\n\tnumNodes: %v\n\tnodeType: %v\n\tregion: %v\n\tone: %v\n\tproject: %v\n\tregionEnv: %v\n\tbackupRegionEnv: %v",
```

Comment:
Why not showing the addons in the message?

Reply:
overlooked, added. Thanks